src/SqlPersistence.Tests/Outbox/OutboxPersisterTests.cs

```diff
@@ -161,6 +161,26 @@ public void StoreAndGet()
         Approver.Verify(result);
     }
 
+    [Test]
+    public async Task TransactionScope()
```

Comment:
Do we need both tests? I think that ATT is good enough.

Reply:
I'm in favor of having both layers. One exercising the component behavior and one exercising the user facing behavior